tests/test_watcher.py

```diff
@@ -0,0 +1,23 @@
+import os.path
+import functools
+
+from lektor import utils
+from lektor import watcher
+
+
+def test_is_interesting(env):
+    cache_dir = os.path.abspath(utils.get_cache_dir())
```

Comment:
Wouldn't this create a directory on the filesystem? Even if I use `os.path.abspath`, I don't write anything during the test.